src/Persistence/Yaml/INamedObject.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
+
+public interface INamedObject
+{
+    string Name { get; set; }
```

Comment:
nit: if possible, I'd remove the setter from the interface (unless strictly necessary)

Reply:
I need to call set during serialization. If you find a way to improve feel free to do it